doc/comments.md

```diff
@@ -0,0 +1,98 @@
+# Comments migration
+
+Hugo being a static website generator, it doesn't process server-side user interactions, which means readers won't be able to leave comments. If you want to retain this abitily, you will need to insert comments and commenting forms through Javascript from a third-party service that will load comments client-side with AJAX.
+
+In case you don't want to retain the ability to add new comments, but still want to migrate old comments, WP2Hugo will import all existing comments for all post types (including attachments, custom post types, etc.) into `/data/comments.yaml`, like so:
+
+```yaml
+- id: "101"
+  author_name: Me
+  author_email: some.user@gmail.com
+  author_url: "https://me.com"
+  published: 2017-04-08T10:29:35Z
+  parent_id: "0"
+  content: excellent, merci.
+  post_url: /relative/path/to/post/
+  post_id: "309"
+- id: "160"
+  author_name: You
+  author_email: other.user@gmail.com
+  author_url: ""
+  published: 2017-04-09T10:29:35Z
+  parent_id: "0"
+  content: cool
+  post_url: /relative/path/to/post/
+  post_id: "309"
+```
+
+From there, you can insert old comments into your pages by adding the following snippet into your theme's `single.html` template:
```

Comment:
Nit: I would suggest moving this HTML into a single file similar to files here https://github.com/ashishb/wp2hugo/tree/main/src/wp2hugo/internal/hugogenerator/hugopage

Then one can just import that page into `single.html`